scripts/install/install-risingwave.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -e
+
+OS=$(uname -s)
+ARCH=$(uname -m)
+VERSION=$(curl -s https://api.github.com/repos/risingwavelabs/risingwave/releases/latest | jq '.tag_name' -r)
```

Comment:
I am afraid that `jq` is not always available on user's env. Any alternatives? (how about https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8)

Reply:
True, I will just grep it.